### DISCUSSION TITLE
Home Assistant error: [hahomematic.client.json_rpc] GET_ALL_SYSTEM_VARIABLES failed: ValueError [invalid literal for int() with base 10: ''] Failed to parse SysVar Alarm messages

### DISCUSSION THREAD

**Discussion by theorician** [SukramJ/aiohomematic#1590] [General]
Hi,

Ich habe gerade mit diesem Fehler zu kämpfen:

[hahomematic.client.json_rpc] GET_ALL_SYSTEM_VARIABLES failed: ValueError [invalid literal for int() with base 10: ''] Failed to parse SysVar Alarm messages 

Hardware: CCU3 mit RaspberryMatic version 3.75.7.20240601. 
Home Assistant v2024.6.1 mit custom_homematic 1.62.1

Alles ziemlich frisch eingerichtet und ohne "customisations":
* CCU3 upgrade auf RaspberryMatic, dann System Reset,
* Home Assistant gestern komplett frisch aufgesetzt.

RaspberryMatic läuft tatsächlich zum Teil ziemlich träge - speziell in der Zeit wenn Home Assistant startet. Die custom_homematic Integration braucht auch mehrere Minuten bis sie startet. Entities werden auch nur selten und langsam in HA updated.

Ich habe keine Ahnung wo ich hier das Bugfixing beginnen soll. Ich habe erstmal den SysVar Scan in den Options disabled. Im GUI kann man (bzw ich) übrigens die Option nicht disablen - sie wird nicht gespeichert und kommt nach einem neustart wieder. Ich habe HA dann gestoppt und das Setting in den config files selbst geändert. Hat leider nichts gebracht.

Hat jemand eine Ahnung?

Ich muss noch dazufügen dass das früher (vor ca. einem Jahr als ich das ganze Gespann hier zuletzt nutzte) immer ziemlich zackig war - mit stock CCU firmware und ein paar Versionen zurück.

**Comment by kramer74** [comment]
Hi,
es kann daran liegen, dass die Systemvariable nicht richtig initialisiert wurde. Da du das System ja neu aufgesetzt hast, hattest du noch keine Alarmmeldungen. Daher vermute ich, dass die Systemvariable einen undefinierten Zustand hat, also nicht 0 ist sonders so etwas wie 'null' hat.
